templates/index.html

```diff
@@ -296,12 +296,14 @@
             <br>
           </div>
 
+
+        
           <div class="col-md-06">
             <div class="form-group reducespace2">
               <label class="heading" id="upload">Upload Desktop Wallpaper</label>
               <label id="browse" class="btn btn-primary btn-block mx-auto">
                 <span data-toggle="tooltip" title=" REC size (1920X1080) or (1280X720)">Browse</span>
-                <input type="file" name="myImage" id="file" class="file form-control" title="Default Wallpaper Image" 
+                <input type="file" name="file" id="file" class="file form-control" title="Default Wallpaper Image" 
```

Comment:
does not look good, there are 0..3 different files that could be uploaded by a user.

Reply:
do you mean that I should revert the changes to myImage

Reply:
I see no reason to change all of them to "file".

Reply:
But if I don't change all of the name attributes to file;
Then the flask will not store the input values in the file array; because the form will store them in the file array
And, I will get an empty array as in issue #300,

Reply:
"will not store the input values in the file array because the form will store them in the file array"

???

Reply:
Yeah the value in the name attribute is only the name of the array in which the different files will get submitted so in this solution I has to keep the name same as I needed all files in the same array

Reply:
Have you tested for any size limitations of that structure?

Reply:
What happens if just the 1 and 3 contan uploads and 2 is empty as opposed to 1 empty and 2 and 3 empty. file[n] then be nil if no input is given or will it results in file[1] sth, file [2] sth file[3] "nil"?

Reply:
No I have not used the index here to loop through the array; it will all the files that the user is uploaded using the web app and save it in the uploads folder